### PR TITLE
Readme file updated.

### PR DESCRIPTION
**Pull request by mosihere**
### List of Changes:

1. Technologies Part completed.
2. License Part completed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,12 @@
 
 ## Technologies
 
+- Python 3
```
Comment (zamaniamin) [inline README.md:38]: 3.11.2
becuse django 4.2 now just supports python 3.8 and up

File: README.md [hunk 2]
```diff
@@ -35,6 +35,12 @@
...
+- Django 4
```
Comment (zamaniamin) [inline README.md:39]: Django 4.2

File: README.md [hunk 3]
```diff
@@ -35,6 +35,12 @@
...
+- DRF (Django Rest Framework)
```
Comment (zamaniamin) [inline README.md:40]: Please add the used version number in requirements.txt file

File: README.md [hunk 4]
```diff
@@ -35,6 +35,12 @@
...
+- Postgres(Database)
```
Comment (zamaniamin) [inline README.md:41]: version for postgres is 15.2

**Review by zamaniamin — COMMENTED** [review]
add version numbers

File: README.md (hunk 3 above)
Comment (mosihere) [reply README.md]: dear Amin, a new commit submitted, review again please.

**Review by zamaniamin — APPROVED** [review]
thanks

**Pull request merged**